operator/v1/types_ingress.go

```diff
@@ -1030,6 +1032,17 @@ type IngressControllerHTTPHeaderBuffer struct {
 	// +kubebuilder:validation:Minimum=4096
 	// +optional
 	HeaderBufferMaxRewriteBytes int32 `json:"headerBufferMaxRewriteBytes,omitempty"`
+	// threadCount defines the number of threads created per ingress controller
+	// pod. Creating more threads allows each ingress controller pod to handle
```

Comment:
Maybe we should be more specific so that people familiar with the implementation don't think this applies to the openshift-router process:
```suggestion
	// threadCount defines the number of threads created per HAProxy process.
	// Creating more threads allows each ingress controller pod to handle
```

Reply:
I was resistant to explicitly saying HAProxy process, but I agree that it's an important detail here. It should be fixed in the latest commit.